docs/user-guide/how_to_use_ssh_plugin.md

```diff
@@ -0,0 +1,119 @@
+# Volcano Job Plugin -- SSH User Guidance
+
+## Background
+**SSH Plugin** is designed for the login without password for pods within a volcano job , which is necessary for workloads 
+such as [MPI](https://www.open-mpi.org/). It often works with `SVC` plugin.
+
+## Key Points
```

Comment:
I think you don't mention that if no arguments are specified, `volcano` would help generate a secret & public key pair for convenience(If I understand the code correctly); also I'm not sure which way is recommended by `volcano`, can you leave a piece of advice for the end-user?

Reply:
Thanks for the advice. I've give the recommand way in the `Key Ponits` chapter and please take a look. But I think it is right to introduce that Volcano will help generate a secret& public key pair by default. Let me add it.

Reply:
Thanks for the advice. I've give the recommand way in the `Key Ponits` chapter and please take a look. But I think it is right to introduce that Volcano will help generate a secret& public key pair by default. Let me add it.